build.yaml

```diff
@@ -1923,6 +1923,7 @@ steps:
     config: grafana/deployment.yaml
     scopes:
       - deploy
+      - test
```

Comment:
Was there a reason for not including this scope initially? Maybe something about needing disks?

Reply:
My guess is that this was removed along with the `test` scope for prometheus, which was removed iirc because of it needs an entire k8s node for itself. But based off this PR run it looks like grafana doesn't need prometheus to exist for it to successfully start. We might even be able to deploy prometheus in tests too and just change the resource requirements for test deploys, since it'll be starting from scratch and not have to load a ton of data in PR runs. I can try that out too if you want.

Reply:
hm going back in the git history it looks like this just never had the `test` scope

Reply:
I'm all for testing grafana in CI, but I do want to be mindful of resources. If you can give it sensible parameters in testing, then this seems fine to me. If we don't like the behavior, then we can revert.

Reply:
It's requesting a modest amount of resources, and its disk is only 10Gi. I think it should have a negligible impact on our testing resources. Prometheus however gives me more pause, since it accumulates a lot of data quickly and I have yet to test how modest we can be within the timeline that the PR deployment would live. Maybe I'll save testing prometheus for the new year and just stick to grafana for now?

Reply:
that seems fine.